pygotham/frontend/templates/talks/proposal.html

```diff
@@ -13,6 +13,9 @@
 
         <div class="row">
           {{ wtf.horizontal_field(form.name, placeholder='Title') }}
+          <small>
+            Got a Co-Presenter? <a href="mailto:organizers@pygotham.org" >Let us know!</a>
```

Comment:
When editing a talk, should we list the co-presenters? As it stands, a user may send an email several times to request the same co-presenters.